go/plugins/googleai/googleai_test.go

```diff
@@ -141,14 +141,23 @@ func TestLive(t *testing.T) {
 		out := ""
 		parts := 0
 		g := googleai.Model(generativeModel)
-		_, err = ai.Generate(ctx, g, req, func(ctx context.Context, c *ai.GenerateResponseChunk) error {
+		final, err := ai.Generate(ctx, g, req, func(ctx context.Context, c *ai.GenerateResponseChunk) error {
 			parts++
 			out += c.Content[0].Text
 			return nil
 		})
 		if err != nil {
 			t.Fatal(err)
 		}
+		out2 := ""
+		for _, c := range final.Candidates {
```

Comment:
I'm not sure this is right. IIRC, candidate chunks are streamed in parallel and can be streamed in any order so it's not necessarily going to be like aaabbbccc.

Reply:
There's only one candidate in this test, so I think it doesn't matter.
I'll just use Candidates[0] instead of looping on the Candidates slice.